olive/model.py

```diff
@@ -165,6 +163,8 @@ def load_model(self) -> onnx.ModelProto:
         return onnx.load(self.model_path)
 
     def prepare_session(self, inference_settings: Dict[str, Any], device: Device):
+        import onnxruntime as ort
```

Comment:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/util/lazy_loader.py
https://github.com/python/peps/blob/main/pep-0690.rst

May `lazy load/import` will be more suitable? Look like lazy load can avoid this kinda duplicated code snippet.

Reply:
I will start a new PR for lazy import